tools/internal_ci/macos/grpc_build_artifacts.sh

```diff
@@ -27,6 +27,7 @@ export PREPARE_BUILD_INSTALL_DEPS_RUBY=true
 export PREPARE_BUILD_INSTALL_DEPS_PHP=true
 source tools/internal_ci/helper_scripts/prepare_build_macos_rc
 
+# TODO(jtattermusch): cleanup this prepare build step (needed for python artifact build)
 # install cython for all python versions
```

Comment:
Unrelated: do you think we should break this job into multiple ones to speed up?

Reply:
I think so and it's my plan eventually but it's going to be a multi-step process. 
(well, eventually this job is going to be obsolete, since we're going to have per-language jobs that do both building the artifacts and test them on the spot).